tests/Features/Types/BladeViewType.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Tests\Features\Types;
+
+class BladeViewType
+{
+    public function testBladeView(): void
+    {
+        $this->doSomethingWithAView('home');
+        $this->doSomethingWithAView('emails.orders.shipped');
+        $this->doSomethingWithAView('users.index');
```

Comment:
I would really like to have tests that can assert that an error is thrown when the string does not point to an existing view. However, due to how the tests are currently set up, this is quite difficult to do. I'm not sure if I should change the testing set up a bit or just leave this as it is.

Reply:
That is a bigger case than this PR. It needs a huge refactoring I guess. We can discuss it in a separate issue or PR if you want to start refactoring.